{"name": "test_failed_new_users[payload2]", "status": "passed", "steps": [{"name": "Creating user with invalid credentials", "status": "passed", "start": 1742461769833, "stop": 1742461769841}, {"name": "Checking response status is 400", "status": "passed", "start": 1742461769841, "stop": 1742461769841}], "parameters": [{"name": "payload", "value": "({'username': 'problem_user1', 'password': 'secret_sauce'}, True)"}], "start": 1742461769832, "stop": 1742461769841, "uuid": "1801d44f-1b9f-419e-a852-3a0c6f0a4969", "historyId": "0392b2392117a83931684fa6d373cb7d", "testCaseId": "737231c88290cb7af71acf5305d104b9", "fullName": "tests.api.tests.test_full_cycle#test_failed_new_users", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Users"}, {"name": "story", "value": "Create invalid users"}, {"name": "parentSuite", "value": "tests.api.tests"}, {"name": "suite", "value": "test_full_cycle"}, {"name": "host", "value": "a978341974be"}, {"name": "thread", "value": "19-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.tests.test_full_cycle"}]}